src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBaseOfT.Tests.cs

```diff
@@ -287,6 +287,14 @@ public async Task ReturnsNonEmptyNativeBoundingBounds(int size)
 			{
 				// https://github.com/dotnet/maui/issues/11020
 			}
+
+			// This type name check is gross, but this project doesn't know anything about Frame/FrameStub
```

Comment:
The frame tests originate from here `FrameHandlerTest.Android.cs` so you could override this test there to provide context in a less-ish gross way

Reply:
Updated.